operator-wg/whitepaper/01_Introduction.md

```diff
@@ -12,6 +12,14 @@ An operator allows for automation of common processes as well as reactive applic
 ## Introduction
 **(Current) Issue: https://github.com/cncf/sig-app-delivery/issues/34**
 
+This white paper defines Operators in a wider context than Kubernetes. It describes their characteristics and components, gives an overview of common patterns currently in use and explains how they differ from Kubernetes controllers.
+
+It also dives deep into their capabilities such as backup, recovery and automatic configuration tuning, gives insight into frameworks currently in use, lifecycle management, security risks and use cases.
```

Comment:
Question on capabilities: What do you think are the most interesting capabilities so I can highlight them here instead of backup, recovery and autoconfig tuninng?

Reply:
Candidates: 

- Provisioning infrastructure
- Day two operations
- auto remediation